src/_footer.scss

```
@@ -1,5 +1,3 @@
-$gray-footer: #fcfcfc !default;
-
 .footer {
-  background-color: $gray-footer;
+  background-color: var(--pgn-color-light-100);
```

Is it possible to do this in a non-breaking way by [utilizing fallbacks](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_cascading_variables/Using_CSS_custom_properties#defining_fallbacks_in_the_var_function)?

I think with fallbacks we'd be able to keep `$gray-footer` and have

```suggestion
  background-color: var(--pgn-color-light-100, $gray-footer);
```

---

I made the change and is working  with paragon 23 and 22

---

Perfect! Once `dist` is re-`.gitignore`d this PR should be good to go!

---

Dist folder removed and gitignore restored.